test/web-platform-tests/to-upstream/html/browsers/the-window-object/window-properties-dont-upstream.html

```diff
@@ -21,23 +21,36 @@
 <script>
 function assert_data_propdesc(pd, Writable, Enumerable, Configurable) {
   assert_equals(typeof pd, "object");
-  assert_equals(pd.writable, Writable);
-  assert_equals(pd.enumerable, Enumerable);
-  assert_equals(pd.configurable, Configurable);
+  assert_equals(pd.writable, Writable, "pd.writable");
+  assert_equals(pd.enumerable, Enumerable, "pd.enumerable");
+  assert_equals(pd.configurable, Configurable, "pd.configurable");
 }
 function assert_accessor_propdesc(pd, hasSetter, Enumerable, Configurable) {
   assert_equals(typeof pd, "object");
-  /*
-  // Something is making window to lose the getters/setters
-  assert_equals(typeof pd.get, "function");
+  assert_equals(typeof pd.get, "function", "typeof pd.get");
   assert_true("set" in pd,
               "Should always have a setter property on the property descriptor");
-  assert_equals(typeof pd.set, hasSetter ? "function" : "undefined");
-  */
-  assert_equals(pd.enumerable, Enumerable);
-  assert_equals(pd.configurable, Configurable);
+  assert_equals(typeof pd.set, hasSetter ? "function" : "undefined", "typeof pd.set");
+  assert_equals(pd.enumerable, Enumerable, "pd.enumerable");
+  assert_equals(pd.configurable, Configurable, "pd.configurable");
 }
 
+var incorrectlyImplementedAsDataProperties = [
```

Comment:
Could you leave a comment here saying that this is jsdom-specific?

Reply:
I’d rather do: <https://github.com/jsdom/jsdom/pull/3139>